src/+otp/Problem.m

```diff
@@ -116,21 +111,8 @@
     end
     
     methods (Access = protected)
-        function validateNewState(obj, newTimeSpan, newY0, newParameters)
+        function validateNewState(~, ~, ~, ~)
```

Comment:
I assume we don't actually need this function now?

Reply:
Subclasses might need to override it.  For example pendulum can show warning if number of masses is different than number of bob lengths.

Reply:
Hmm, I agree, but, then should it be abstract? Would that break anything?

Reply:
The vast majority of problems don't need to do validation (past what `Problem` already does).  Adding abstract would make these problems implement empty functions.

Reply:
Makes sense.